include/nuttx/video/fb.h

```diff
@@ -1106,7 +1109,30 @@ int fb_register_device(int display, int plane,
  *
  ****************************************************************************/
 
-int fb_register(int display, int plane);
+static inline_function unused_code int fb_register(int display, int plane)
```

Comment:
```suggestion
static inline_function int fb_register(int display, int plane)
```

Reply:
without unused_code, there is a compilation error in macOS(sim-01) like Error: nxglib/nxglib_setpixel_1bpp.c:273:50: error: unused function 'fb_register' [-Werror,-Wunused-function]